File: android/src/main/java/com/valentingrigorean/arcgis_maps_flutter/map/MarkerControllerSink.java
package com.valentingrigorean.arcgis_maps_flutter.map;

public interface MarkerControllerSink extends GraphicControllerSink {
    void setSelectedScale(float selectedScale);

    void setIcon(BitmapDescriptor bitmapDescriptor);

    void setBackground(BitmapDescriptor bitmapDescriptor);

    void setIconOffset(float offsetX,float offsetY);

    void setOpacity(float opacity);

    void setAngle(float angle);
}

File: android/src/main/java/com/valentingrigorean/arcgis_maps_flutter/map/MarkerControllerSink.kt
package com.valentingrigorean.arcgis_maps_flutter.map

interface MarkerControllerSink : GraphicControllerSink {
    fun setSelectedScale(selectedScale: Float)
    fun setIcon(bitmapDescriptor: BitmapDescriptor?)
    fun setBackground(bitmapDescriptor: BitmapDescriptor?)
    fun setIconOffset(offsetX: Float, offsetY: Float)
    fun setOpacity(opacity: Float)
    fun setAngle(angle: Float)
}